processor/processor.go

```diff
@@ -232,7 +232,7 @@ func (proc *HandleT) newDestinationTransformationStat(sourceID, workspaceID, tra
 	numEvents := proc.statsFactory.NewTaggedStat("proc_num_dt_input_events", stats.CountType, tags)
 	numOutputSuccessEvents := proc.statsFactory.NewTaggedStat("proc_num_dt_output_success_events", stats.CountType, tags)
 	numOutputFailedEvents := proc.statsFactory.NewTaggedStat("proc_num_dt_output_failed_events", stats.CountType, tags)
-	destTransform := proc.statsFactory.NewTaggedStat("proc_dest_transform", stats.TimerType, tags)
+	destTransform := proc.statsFactory.NewTaggedStat("proc_dt_stage_req_time", stats.TimerType, tags)
```

Comment:
I am not sure if we can simply rename them, I believe there is a dashboard using `proc_dest_transform`

Reply:
it is only being used here: https://github.com/rudderlabs/rudderstack-operator/blob/c19fd4a0cf1fad527190484b49eb267e4f87ce9d/helm-charts/rudderstack-monitor/charts/grafana/dashboards/rudderstack-metrics-dashboard.json#L3389

@chandumlg Can simply change it here too? This will makes things more readable.

Reply:
Looking at this again, the `req` in `proc_dt_stage_req_time` might be a bit miss leading since we are not measuring the time a single request takes but rather the whole destination transformation stage. I would suggest `proc_dt_stage_duration`

Also since we are changing this, wondering if it would be better if we had a single metric `proc_transform_stage_duration` with tag `type` (destination or user)

Reply:
`proc_transform_stage_duration` 👍 

Use both source id and destination id in the tags.

Reply:
modified